### ISSUE TITLE
What is the FPS per second?

### ISSUE THREAD

**Issue by balavenkatesh3322** [whitphx/streamlit-webrtc#242]
What is the FPS per second?

How can I change the FPS rate in the application?

**Comment by whitphx** [comment]
It's not specified by default.

You may be able to control it via `media_stream_constraints media_stream_constraints` (ex: https://github.com/whitphx/streamlit-webrtc/blob/3ace08de428af2be262219fd248560371346bf6d/app.py#L84).
For now, this parameter is used as an argument of [`navigator.mediaDevices.getUserMedia()`](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) on the frontend to initialize the media devices, and `video.frameRate` can be set like [this](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#frame_rate).

How is code like below though I have not tested it yet?
```python
WEBRTC_CLIENT_SETTINGS = ClientSettings(
    rtc_configuration={"iceServers": [{"urls": ["stun:stun.l.google.com:19302"]}]},
    media_stream_constraints={
        "video": {
            "frameRate": { "ideal": 10, "max": 15 }
        },
        "audio": True,
    },
)
```

**Comment by whitphx** [comment]
Sorry, it's not supported yet.
Wait for the next release.

**Comment by whitphx** [comment]
It's available on v0.21.0
